src/components/modals/ModalCloseButton.tsx

```diff
@@ -0,0 +1,20 @@
+import { IoClose } from 'solid-icons/io'
+import { Component, Setter } from 'solid-js'
+
+type Props = {
+  setShouldDisplay: Setter<Boolean>
+}
+const CloseButton: Component<Props> = ({ setShouldDisplay }) => {
```

Comment:
Components should have the same name as the file.
```suggestion
const ModalClose: Component<Props> = ({ setShouldDisplay }) => {
```